src/services/user.service.ts

```diff
@@ -0,0 +1,93 @@
+import UserModel from '../models/user.model';
+import { UserInterface } from '../interfaces/user.interface';
+import { userModelInstance } from './../models/user.model';
+import * as userInfo from './../data/users.json';
+import { Model, Op} from 'sequelize';
+
+
+const defaultUser = {
+  "login": "petrenckira",
+  "password": "password123",
+  "age": 22,
+  "isDeleted": false
+};
+
+export default class UserService {
+
+  constructor(public userModel: any) {
+  }
+
+  async createDefaultUser(): Promise<Model> {
```

Comment:
I hope this one should be wherever else but not in UserService.
You should look at https://medium.com/@shannen.ye/setting-up-a-database-and-seed-file-7e73fe2a9fe6